conda-store-server/conda_store_server/orm.py

```diff
@@ -313,6 +314,11 @@ def conda_pack_key(self):
     def docker_manifest_key(self):
         return f"docker/manifest/{self.build_key}"
 
+    @property
+    def constructor_installer_key(self):
+        ext = "exe" if sys.platform == "win32" else "sh"
+        return f"installer/{self.build_key}.{ext}"
```

Comment:
Note for reviewers: this is similar to other keys in this file

Reply:
We can also have PKG installers on macOS. Is this handled?

Reply:
> We can also have PKG installers on macOS. Is this handled?

It's handled by constructor, but this PR generates .sh on Linux and macOS. .sh is the default installer type on Linux and macOS in constructor.

It makes everything either if you have one installer per platform (less things to test, one endpoint for all installers on the API side). Do you think it's still worth it to have two installers on macOS?

Reply:
We don't _need_ it but at some point I am sure someone will end up asking for them (they can be branded, they feel more "OS-native", etc). Just wanted to make sure this assumption was made knowing there's a non-sh installer on macOS.